rust/src/lib/ifaces/bond.rs

```diff
@@ -1282,3 +1346,46 @@ impl MergedInterface {
         Ok(())
     }
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Default)]
+#[serde(rename_all = "kebab-case")]
+#[non_exhaustive]
+pub struct BondPortConfig {
+    /// name is mandatory when specifying the ports configuration.
+    pub name: String,
+    #[serde(
+        skip_serializing_if = "Option::is_none",
+        default,
+        deserialize_with = "crate::deserializer::option_i32_or_string"
+    )]
+    pub prio: Option<i32>,
```

Comment:
Please use `priority` instead of `prio`. This user facing, abbreviation is bad idea.

Reply:
And document please.